okapi-core/src/test/java/org/folio/okapi/managers/InternalModuleTest.java

```diff
@@ -69,4 +72,34 @@ void pathNotFound(String path, VertxTestContext vtc) {
         }));
   }
 
+  /**
+   * Check that invalid JSON in body is ignored.
+   */
+  @ParameterizedTest
+  @CsvSource({
+    "GET,  /_/proxy/health",
+    "GET,  /_/version",
+  })
+  void ignoreBody(HttpMethod httpMethod, String path, VertxTestContext vtc) {
+    internalService(httpMethod, path, "}").onComplete(vtc.succeedingThenComplete());;
+  }
+
+  /**
+   * Check that invalid JSON in body is caught.
+   */
+  @ParameterizedTest
+  @CsvSource({
+    "POST, /_/proxy/import/modules",
+    "GET,  /_/proxy/modules",
```

Comment:
Interesting. [Okapi RAML](https://github.com/folio-org/okapi/blob/v4.8.2/okapi-core/src/main/raml/okapi.raml#L432-L483) did not specify request body for this GET request.